domain/schema/controller.go

```diff
@@ -52,6 +54,10 @@ func ControllerDDL() *schema.Schema {
 		userSchema,
 		modelConfigDefaults,
 		flagSchema,
+		annotationSchemaForTable("application"),
+		annotationSchemaForTable("machine"),
+		annotationSchemaForTable("unit"),
+		annotationSchemaForTable("model"),
```

Comment:
The storage tables are now there under https://github.com/juju/juju/pull/16682. That should leave only `charm` outstanding.